app/src/main/java/com/finite/gdscphcet/ui/HomeFragment.kt

```diff
@@ -129,9 +129,12 @@ class HomeFragment : Fragment() {
                         requireContext(),
                         upcomingEventsList.reversed() as MutableList<UpcomingEventModel>
                     )
+                    upcomingShimmer.stopShimmer()
                     upcomingShimmer.visibility = View.GONE
 
                 } else{
+                    upcomingShimmer.stopShimmer()
```

Comment:
Bro I not think so here is it needed as when data loaded and there is no upcoming event then it show placeholder for no upcoming event.

After that is it working fine ??

Reply:
Yes it is working fine, checked after making the changes